packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/iOS/signin.md

```diff
@@ -1,17 +1,51 @@
-Login events can be triggered by `@IBAction`, `viewDidLoad`, or programmatically. To launch the event, first update your `AppDelegate` to include the following function to allow the redirect to occur:
+Before using the SDK you have to create a new object of OktaOidc class. You can instantiate OktaOidc w/o parameters that means that SDK will use Okta.plist for configuration values. Alternatively you can create OktaOidc instance with custom configuration.
 
-Code sample
+```swfit
+import OktaOidc
 
-Then, you can start the authorization flow by simply calling `login`:
+// Use the default Okta.plist configuration
+let oktaOidc = OktaOidc()
 
-Code sample
+// Use configuration from another resource
+let config = OktaOidcConfig(/* plist */)
+// or: let config = OktaOidcConfig(/* dictionary */)
 
-## Handle the Login State
+// Instantiate OktaOidc with custom configuration object
+let oktaOidc = OktaOidc(configuration: config)
+```
 
-In native applications, it is common for users to have a long-lived session. It is important for the app to manage the user's session by refreshing tokens when they expire, using the `refresh_token` or re-prompting the user to sign in.
+Then, you can start the authorization flow by simply calling `signInWithBrowser`. In case of successful authorization, this operation will return valid `OktaOidcStateManager` instance in its callback:
```

Comment:
Then, you can start the authorization flow by simply calling `signInWithBrowser`. When successfully authorized, this operation returns a valid `OktaOidcStateManager` instance in its callback: